src/components/LandingPage/LandingPage.js

```diff
@@ -263,17 +264,31 @@ const JoinActionArrow = styled.img`
 
 const LandingPage = () => {
 
-    const NavItem = withRouter(
+    const BlueButton = withRouter(
         ({ option, route, history, location, children }) => {
-            return (
-                <DarkButton
-                    onClick={() => {
-                        history.push(route);
-                    }}
-                >
-                    {children}
-                </DarkButton>
-            );
+                return (
+                    <DarkButton
+                        onClick={() => {
+                            history.push(route);
+                        }}
+                    >
+                        {children}
+                    </DarkButton>
+                );
+        }
+    );
+
+    const NormalButton = withRouter(
```

Comment:
Instead of having one function for NormalButton and one for DarkButton, we could have one function and also one button component and pass the css properties that are different into the button component and then in the button component above we would have a few properties like this: `color: ${(props) => props.color};`

Fine to merge this now though.